backend/src/fun_forecast_backend/weather/weather.py

```diff
@@ -9,11 +9,30 @@
 
 from fun_forecast_backend.core.standard_logger import get_logger
 from fun_forecast_backend.shared.dataclasses import WeatherCondition, HourData
-from fun_forecast_backend.weather.dataclasses import WeatherApiPoint, WeatherData, Period
+from fun_forecast_backend.weather.dataclasses import WeatherApiPoint, WeatherData, Period, LatLongResponse
 
 logger = get_logger()
 
 
+def get_location_by_lat_and_long(lat: float, long: float) -> LatLongResponse:
```

Comment:
nit
```suggestion
def get_location_by_coords(lat: float, long: float) -> LatLongResponse:
```
Also I understand this uses `point_from_coords` which lives in weather, but maybe this *and* point_from_coords actually lives in location.py? IDK, the nature of code bends toward 🍝

Reply:
Yeah it doesn't have a natural fit. Definitely spaghetti. Let me see....